cmd/Dockerfile

```diff
@@ -3,12 +3,13 @@ FROM golang:1.9-alpine as builder
 
 RUN apk add --no-cache make gcc musl-dev linux-headers
 
+VOLUME ["/configs"]
```

Comment:
這其實是多餘的。這會 新增一個persistent volume和host共享，但docker-compose那邊的寫法是mount一個已經存在的目錄到container中

Reply:
Do we only need to consider `docker-compose`? I am thinking that we can also support `docker` directly. E.g., `docker run -v configs:/configs eth-indexer`

It looks redundant with `docker-compose`, but it can actually be independent with `docker`. My suggestion will be keeping this line.

Reply:
After discussed with @bailantaotao about our own deployment, we will not use this volume. To prevent resource wasted, I will remove this line. Thanks!!

Reply:
thanks for the feedback! updated in d5996ec69017544c648f593a55b1ad00970db447